### PR TITLE
fix(css-loader): upgraded to latest css-loader api as of 3.4.2

### PR DESCRIPTION
**Pull request by mtomcal**
- Upgraded the internal css-loader replacements going from css-loader `3.3.0` to latest as of now, `^3.4.2`
CSS Loader in `3.3.0` inserted the following into the bundle:
```js
exports = module.exports = __webpack_require__(/*! ./node_modules/css-loader/dist/runtime/api.js */ "./node_modules/css-loader/dist/runtime/api.js")(false);
// Module
exports.push([module.i, "aside.my-root__OfflineWarning__offlineWarning___3qufd {\n  position: fixed;\n  width: 100%;\n  z-index: 100 !important;\n}\n\n.my-root__OfflineWarning__offlineFontFetcher___3IL4P {\n  position: absolute;\n}\n.my-root__OfflineWarning__offlineFontFetcher___3IL4P:before {\n  content: \" \";\n}", ""]);
// Exports
exports.locals = {
	"offlineWarning": "my-root__OfflineWarning__offlineWarning___3qufd",
	"offlineFontFetcher": "my-root__OfflineWarning__offlineFontFetcher___3IL4P"
};
```

CSS Loader in `3.4.2` inserts the following into the bundle today:
```js
// Imports
var ___CSS_LOADER_API_IMPORT___ = __webpack_require__(/*! ./node_modules/css-loader/dist/runtime/api.js */ "./node_modules/css-loader/dist/runtime/api.js");
exports = ___CSS_LOADER_API_IMPORT___(false);
// Module
exports.push([module.i, ".my-root__ErrorLayout__ErrorLayout___2B91F {\n  margin: 0 auto;\n}\n\n.my-root__ErrorLayout__main___2IftH {\n  max-width: 1000px;\n  width: 100%;\n  margin: 0 auto;\n  padding: 10px;\n}", ""]);
// Exports
exports.locals = {
	"ErrorLayout": "my-root__ErrorLayout__ErrorLayout___2B91F",
	"main": "my-root__ErrorLayout__main___2IftH"
};
module.exports = exports;
```

**@10xLaCroixDrinker is required to review and approve this before merging**

### CODE REVIEW

**Comment by mtomcal** [comment]
Manually validated that the html document served by One App outputs `.ssr-css` block when serving a module using the changes I made to bundler.

File: packages/one-app-bundler/package.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
     "babel-preset-amex": "^3.2.0",
     "chalk": "^2.4.2",
     "core-js-compat": "^3.4.2",
-    "css-loader": "3.3.0",
+    "css-loader": "^3.4.2",
```
Comment (10xLaCroixDrinker) [inline packages/one-app-bundler/package.json]: we should keep this locked down in case this changes again

Comment (mtomcal) [reply packages/one-app-bundler/package.json]: Fixed in d6df64e

File: packages/one-app-bundler/__tests__/webpack/ssr-css-loader/__snapshots__/index.spec.js.snap [hunk 2]
```diff
@@ -10,24 +10,25 @@ exports[`SSR CSS loader should remove css-loader's css-base, replace its push an
...
+  padding: 10px;
+}\\", \\"\\"]);
+      // Exports
       exports = module.exports = {
-      	\\"One\\": \\"react-select__One___hIKe\\",
-      	\\"also\\": \\"react-select__also___8ObpT\\",
+      	\\"ErrorLayout\\": \\"my-root__ErrorLayout__ErrorLayout___2B91F\\",
+      	\\"main\\": \\"my-root__ErrorLayout__main___2IftH\\"
       };
+      module.exports = exports;
```
Comment (Francois-Esquire) [inline packages/one-app-bundler/__tests__/webpack/ssr-css-loader/__snapshots__/index.spec.js.snap]: The source replaces `exports.locals =` with `exports = module.exports =`. Would this last line be needed?

Could `module.export = exports;` be safely removed? Maybe before this:

https://github.com/americanexpress/one-app-cli/blob/bdb322109ecb01a6cb4b0b7936ff87eefb97d44b/packages/one-app-bundler/webpack/loaders/ssr-css-loader/index.js#L37-L40

Comment (mtomcal) [reply packages/one-app-bundler/__tests__/webpack/ssr-css-loader/__snapshots__/index.spec.js.snap]: I tried removing this and the test cases failed. This replacement is definitely being used from what the snapshot diffs tell me.

**Pull request merged**